lighty-examples/lighty-cluster-app/runLocal3NodeCluster.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+cd target
+if [ ! -d  lighty-cluster-app-10.1.1-SNAPSHOT ]; then
```

Comment:
Please, try to add "lighty-cluster-app-10.1.1-SNAPSHOT" to some variable in this script or something similar. It will be easier to bump versions of lighty in the future.